wireguard-install.sh

```diff
@@ -39,7 +39,14 @@ function addClient() {
 	echo "[Interface]
 PrivateKey = $CLIENT_PRIV_KEY
 Address = $CLIENT_WG_IPV4/24,$CLIENT_WG_IPV6/64
-DNS = $CLIENT_DNS_1,$CLIENT_DNS_2
+
+# Fix for bug where trailing comma is left if no secondary DNS specified
+if [ -z '$CLIENT_DNS_2' ]
+then
+	DNS = $CLIENT_DNS_1,$CLIENT_DNS_2
+else
+	DNS = $CLIENT_DNS_1
+fi
```

Comment:
You're inside an `echo` statement here so it won't work :) But the logic is correct

Reply:
Aha, my mistake! Shows what I get for going "ha, that's easy, I don't even need to test this." I'll see if I can come up with an actual fix that isn't absolutely gross. :)